tests/ScheduledTaskSubscriber/ScheduledTaskSubscriberTest.php

```diff
@@ -153,4 +153,26 @@ public function it_works_correctly_when_a_task_is_run_in_the_background()
 
         Bus::assertDispatched(PingOhDearJob::class);
     }
+
+    /** @test */
+    public function it_stores_the_command_output()
+    {
+        TestKernel::replaceScheduledTasks(function (Schedule $schedule) {
+            $path = storage_path('logs/dummy-task.log');
+
+            $schedule
+                ->call(fn () => file_put_contents($path, 'dummy output'))
```

Comment:
It doesn't work like that. When you define an anonymous function you're not in a new PHP process so you can't really output anything. But that `file_put_contents` simulates the output that a process would write.

Reply:
Simulating isn't a test for that. You can schedule a real command class for real output.

Reply:
You can't really run real commands through the scheduler in tests, or at least not that easy. 
Laravel creates a new PHP process that runs your command separately of the test case.  And since  orchestra/testbench doesn't include the `artisan` script you can't run real commands through the scheduler in tests.

But if you look at the Laravel source code for the scheduler and how it processes the output, you'll see that basically the process that runs your command will pipe the output to a file inside the logs folder.

Reply:
@freekmurze any thoughts on this ?

Reply:
@cretueusebiu have you tested it in a fresh Laravel app with the inspire command or any other command with some output?
So far I see it to make the output logging work I have to manually call `storeOutput()` on the scheduled command? So I think that the README needs an update as well. 🤔

Reply:
@Gummibeer  Yes, I've tested it by replacing the package with my fork and it works. Yes you actually have to call `storeOutput()` to store it.

@freekmurze  You mean when will `getEventTaskOutput` return something ? That's when `ScheduledTaskFinished` is dispatched and you called `storeOutput()`.

Reply:
@freekmurze If you think this is more of an edge case or potentially bc, could we at least make the `MonitoredScheduledTask` model configurable ? That way the base model could be extended and just override the `markAsFinished` method with the new implementation.

Reply:
@Gummibeer opened #43 with an actual artisan command to test.